### PR TITLE
Added Currency class and include a field which shows the price in USD

### LINKED ISSUE #1: Currency Conversion
All prices are in Yen, add a USD conversion.

### PR DESCRIPTION
**Pull request by thebunanna**
Approve plz 🙏

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -1,36 +1,39 @@
...
+        genre, _, is_r18, image_preview_url = scrape_url(url)
+
+    # Add embed
+    embed=discord.Embed(
+        url = url,
+        title = f'{title} - {circle_name} ({author_name})',
+    )
+    embed.set_thumbnail(url=image_preview_url)
+    embed.add_field(name="Price (¥)", value=price_in_yen, inline=True)
+    embed.add_field(name="Price ($)", value=price_in_yen * currency.get_rate(), inline=True)
```
Comment (rchhong) [inline bot.py]: I think it would make more sense to move this logic into the currency class instead of out here.

File: currency.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+        parameters = {"api_key": self.api_key, "format": "json", "from" : self.currency_from, "to" : self.currency_to}
+
+        url = "https://api.getgeoapi.com/v2/currency/convert"
+
+        response = requests.get(url, parameters)
+        json = response.json()
+        print(response.json())
+        if json["status"] == "success":
+            self.current_rate = json["rates"]["USD"]["rate"]
```
Comment (rchhong) [inline currency.py]: Need to convert to float first.

File: bot.py (hunk 1 above)
Comment (rchhong) [reply bot.py]: Also, add rounding to nearest cent.

**Review by rchhong — CHANGES_REQUESTED** [review]
There's a bug that breaks the bot + some stuff that needs to be cleaned up.

**Comment by thebunanna** [comment]
this is what happened when I decide to not test with the actual discord bot

File: .github/workflows/main.yml
Comment (rchhong) [inline .github/workflows/main.yml:1]: I'm just going to trust you on this one for now but I'm pretty sure the env field needs to have more parameters

**Review by rchhong — APPROVED** [review]
Looks good to me, will clean up the prints later.

**Pull request merged**